engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -579,10 +579,12 @@
   protected ScriptingEnvironment scriptingEnvironment;
   protected List<ScriptEnvResolver> scriptEnvResolvers;
   protected ScriptFactory scriptFactory;
+  protected String scriptEngineNameJavaScript;
   protected boolean autoStoreScriptVariables = false;
   protected boolean enableScriptCompilation = true;
   protected boolean enableScriptEngineCaching = true;
   protected boolean enableFetchScriptEngineFromProcessApplication = true;
+  protected boolean enableScriptEngineLoadExternalResources = false;
```

Comment:
This flag will enable `polyglot.js.allowIO` on GraalVM JavaScript in general. I am not 100% certain that this will also allow loading from classpath as that might be a different issue with GraalVM JavaScript.
You can enable this flag in  Camunda Platform Run by using the [Generic Properties](https://docs.camunda.org/manual/user-guide/spring-boot-integration/configuration/#generic-properties).

Reply:
> I am not 100% certain that this will also allow loading from classpath

But isn't this exactly what [`shouldLoadExternalScript()`](
https://github.com/camunda/camunda-bpm-platform/pull/1466/files#diff-31b4237e37965ccaa00cdad64e0a044134e101c5a8e6a38422e4d2ac9872b6c5R140) tests? 🤔

Reply:
Well, yes and no. 

It tests that you can load an external resource with a URL, which is supported by GraalVM JavaScript. In this case, this resource sits on the classpath of the test as well, yes. 

I am not a complete expert on this, but I think you can however not reference the resource with a `load('classpath:...')` descriptor, as far as I know. This seems to be an issue in GraalVM JavaScript that we cannot simply enable by switching an option flag.
Therefore: yes and no, depending on how you interpret "loading a script file from the classpath".

I hope that is a reasonable explanation.

Reply:
So does this mean it works if the string passed to `load()` is a `file://` URL?

But that would mean the scripts in the process need to hard code the absolute path of "libraries" on the machine where the engine is running on. This makes reusing script code in multiple processes quite cumbersome. 🙁 

The `classpath:` approach was very clean in this regard: you could deploy your common script libraries onto the classpath and then `load()` them from all processes.

Reply:
I fully agree. 
This is however something that is more related to how GraalVM JavaScript handles this. We will not be able to provide customizations for all aspects that GraalVM JavaScript handles differently. Especially if this is something that is not easily configurable. GraalVM JavaScript provides something like a `load('resource:...')` mechanism. I am not sure though if this can be used as a replacement for `classpath:`.
That being said, with #1468 we will also allow to provide a custom script engine resolver that could provide a GraalVM JavaScript instance that is configured to allow such things. If there is enough traction, it might make sense to create a community extension that provides this functionality as an engine plugin for example.